src/NuGet.Core/NuGet.Packaging/Signing/SignatureLog.cs

```diff
@@ -38,34 +38,14 @@ public static SignatureLog DebugLog(string message)
             return new SignatureLog(LogLevel.Debug, NuGetLogCode.Undefined, message);
         }
 
-        public static SignatureLog InvalidInputError(string message)
+        public static SignatureLog Issue(bool fatal, NuGetLogCode code, string message)
         {
-            return new SignatureLog(LogLevel.Error, NuGetLogCode.NU3001, message);
-        }
-
-        public static SignatureLog InvalidTimestampInSignatureError(string message)
-        {
-            return new SignatureLog(LogLevel.Error, NuGetLogCode.NU3050, message);
-        }
-
-        public static SignatureLog InvalidPackageError(string message)
-        {
-            return new SignatureLog(LogLevel.Error, NuGetLogCode.NU3002, message);
-        }
+            // Warning codes are 500 higher than error codes
+            code = fatal ? code : code + 500;
```

Comment:
Chatted with Justin.  It would be simpler to have one code for each scenario.  I wouldn't hold up this PR though.

Reply:
So have the same error code for warning or error when is the same scenario? (the scenario will only differ on the policy)